app/src/main/java/com/puzzlebench/clean_marvel_kotlin/domain/usecase/StoreCharactersUseCase.kt

```diff
@@ -0,0 +1,9 @@
+package com.puzzlebench.clean_marvel_kotlin.domain.usecase
+
+import com.puzzlebench.clean_marvel_kotlin.domain.contracts.CharacterRepository
+import com.puzzlebench.clean_marvel_kotlin.domain.model.Character
+
+
+open class StoreCharactersUseCase(private val storeCharactersService: CharacterRepository) {
```

Comment:
Why this is an open class and method?

Reply:
This happends in both useCases, pls check it